Classes/Emogrifier.php

```diff
@@ -464,6 +466,85 @@ private function mapAllInlineStylesToHtmlAttributes(\DOMXPath $xPath)
     }
 
     /**
+     * Searches for all nodes with a style attribute and removes the "!important" annotations out of
```

Comment:
The term here is not correct: According to https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#The_!important_exception, `!important` is a rule, not an annotation.

Reply:
According to https://www.w3.org/TR/css3-cascade/#importance - which seems to be part of the official specification of CSS in 2017 (https://www.w3.org/TR/CSS/#css) - it is an annotation. Also, "rule" can be confusing, because it is mainly used as the name for a selector + list of declarations unit. But I do not want to force anything, if you confirm that I should change 'annotation' with 'rule', I'll do it :)

Reply:
Thanks for the link! Then let's keep "annotation". (To me, the W3 specs are indeed a lot more explicit on that it's an annotation, not a rule, than the MDN.)